HW5/hw_5_task_1.py

```diff
@@ -0,0 +1,30 @@
+# Завдання:
+# 1. Створити програму, яка буде очікувати від користувача введення тексту і виведе інформацію по кожному надрукованому символу:
+# це “число” + яке воно (парне, непарне),
+# це “буква” + яка вона (велика чи маленька),
+# це “символ”
+
+text = input("Enter some text ")
+symbol = list(text)
```

Comment:
nit: it is not required to convert to a list. letter is returned by iterating through the sting